### PR TITLE
MySQL - optimize. maybe.

### PR DESCRIPTION
**Pull request by Apogate**
Completed and ready for merging pending the destruction of my dreams.

What it does:

"optimizes" all SQL queries by adding limits where necessary to cut down on execution time/memory usage/whatever. I am not as experienced with SQL as I am, say, with ur mum, so I don't know how much of an impact this will have performance wise. Probably significant on higher loads. maybe this is a huge flaming wreck and I don't even realize it.

### CODE REVIEW

File: _core/admin/delete.php [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+			uses the checkbox deletion method while browsing.
+			
+			targeted actually processes the deletion, deleting the targeted post(s), 
+			their children, reports and destroys any necessary files (HTML and media).
+		
+			It is in desperate need of a rewrite. (meme header)
+		=========================================================================
+*/
+
+
```
Comment (Apogate) [inline _core/admin/delete.php:18]: no

**Comment by RePod** [comment]
Well if it's worth anything, SQL will spend additional time processing the limit syntax.

**Comment by RePod** [comment]
Why do we still need to check the `board` column in the queries? That's just one more useless operation since we're not putting everything in one table.

Setting the column is fine, but that'd probably be best done automatically by MySQL itself as a default value.

**Comment by Apogate** [comment]
no

**Pull request closed without merging**